src/unix/notbsd/mod.rs

```diff
@@ -465,6 +465,7 @@ pub const F_SEAL_SEAL: ::c_int = 0x0001;
 pub const F_SEAL_SHRINK: ::c_int = 0x0002;
 pub const F_SEAL_GROW: ::c_int = 0x0004;
 pub const F_SEAL_WRITE: ::c_int = 0x0008;
+pub const F_SEAL_FUTURE_WRITE: ::c_int = 0x0010;
```

Comment:
Is this available in emscripten ?

Reply:
Not yet, I think. It also seems to be missing from musl currently. What is the proper place to put this in that case? `src/unix/notbsd/linux/other/mod.rs` and a copy to `src/unix/notbsd/android/mod.rs`?

Reply:
So one copy goes into `android/mod.rs`, another copy goes into `linux/mips/mod.rs`/`linux/s390x/mod.rs`, `linux/other/mod.rs` - basically everywhere in `linux` except `linux/musl/mod.rs`.